src/components/legend.ts

```diff
@@ -167,26 +167,15 @@ export module Component {
 
     public _requestedSpace(offeredWidth: number, offeredHeight: number): _SpaceRequest {
       var estimatedLayout = this._calculateLayoutInfo(offeredWidth, offeredHeight);
-      var rowLengths = estimatedLayout.rows.map((row: string[]) => {
+
+      var estimatedRowLengths = estimatedLayout.rows.map((row: string[]) => {
         return d3.sum(row, (entry: string) => estimatedLayout.entryLengths.get(entry));
       });
+      var longestEstimatedRowLength = _Util.Methods.max(estimatedRowLengths, 0);
```

Comment:
Note a big deal, but how much trouble do you think it would be to inline estimatedRowLengths